chromium_src/components/translate/core/browser/translate_language_list.cc

```diff
@@ -13,8 +13,8 @@ namespace translate {
 
 namespace {
 const char* const kBraveDefaultLanguageList[] = {
-    "de", "en", "es", "fr", "hi", "it", "ja",      "nl",
-    "pl", "pt", "ro", "ru", "tr", "vi", "zh-Hans",
+    "de", "en", "es", "fr", "hi", "it", "ja",    "nl",
+    "pl", "pt", "ro", "ru", "tr", "vi", "zh-CN",
```

Comment:
can we also add `"zh-TW"` to default list?

Reply:
@linhkikuchi 
Hm, I can't see that it's supported on `https://translate.brave.com/translate_a/l`